backend/slack_commands/formatters.py

```
@@ -63,12 +63,29 @@ def format_session_list(
+
+    lines = [f"*Your Teams* ({len(teams)} total)\n"]
+
+    for team in teams:
+        name = team.get("name") or team.get("team_id") or "?"
+        team_id = team.get("team_id") or "?"
+        lines.append(f":busts_in_silhouette: *{name}* (`{team_id}`)")
+
+    return SlackResponse(text="\n".join(lines))
+
+
+def format_workflow_list(workflows: list, label: str = None) -> SlackResponse:
```

Ruff (0.15.21)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)